sdk/cosmosdb/cosmos/src/queryExecutionContext/hybridQueryExecutionContext.ts

```diff
@@ -403,6 +404,15 @@ export class HybridQueryExecutionContext implements ExecutionContext {
   }
 
   private replacePlaceholders(query: string, globalStats: GlobalStatistics): string {
+    // Validate globalStats object
+    if (
+      !globalStats ||
+      !globalStats.documentCount ||
+      !Array.isArray(globalStats.fullTextStatistics)
+    ) {
+      throw new Error("Invalid globalStats object provided.");
```

Comment:
Can we replace the error message with "GlobalStats validation failed"

Reply:
done